template-handbook-article.md

```diff
@@ -0,0 +1,147 @@
+<!-- Title, author, date block standard in all files -->
+# Title (ex: How to Use `o1-clone` tool)
```

Comment:
I would just suggest making "Use" lowercase instead, since we want the titles to be in sentence case

Reply:
Also, maybe this document should go in the contributing folder?